packages/flutter_test/lib/src/controller.dart

```diff
@@ -400,6 +400,19 @@ abstract class WidgetController {
     });
   }
 
+  /// A simulator of how the framework handles a serials of [PointerEvent]s
+  /// received from the flutter engine.
+  ///
+  /// The [PointerEventPacket.timeStamp] is used for time delay of an event
+  /// packet relative to the starting point of the method call.
+  ///
+  /// Returns a list of the difference between [PointerEventPacket.timeStamp]
+  /// and the real timestamp when the event packet is processed. The closer
+  /// these values are to zero the more faithful it is to the `records`.
+  ///
+  /// See [PointerEventPacket].
+  Future<List<Duration>> handlePointerEventPack(List<PointerEventPacket> records);
```

Comment:
`handlePointerEventPack` to `handlePointerEventPackets` for the recent renaming, and make it plural to signal that there's a list of packets. `records` may be better names as packets too.